apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/exception/StatusRuntimeExceptionMapper.java

```diff
@@ -24,9 +24,12 @@
 import org.jboss.resteasy.reactive.RestResponse;
 import org.jboss.resteasy.reactive.RestResponse.ResponseBuilder;
 import org.jboss.resteasy.reactive.server.ServerExceptionMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /** Simple exception mapper for the {@link StatusRuntimeException}. */
 public class StatusRuntimeExceptionMapper {
+  private static final Logger LOGGER = LoggerFactory.getLogger(StatusRuntimeExceptionMapper.class);
 
   @ServerExceptionMapper
   public RestResponse<ApiError> statusRuntimeException(StatusRuntimeException grpcE) {
```

Comment:
todo below can be removed

Reply:
Done in https://github.com/stargate/stargate/pull/2171/commits/eb812454b4c804ffb27004f518b852ea9e01ed5b